server/database/dataGenerator.js

```diff
@@ -0,0 +1,90 @@
+const numbers = [];
+for (let i = 1; i < 994; i += 1) {
+  numbers.push(i.toString().padStart(4, '0').concat('.jpg'));
+}
+
+const exceptions = ['0013.jpg', '0376.jpg', '0485.jpg', '0566.jpg', '0686.jpg', '0718.jpg', '0775.jpg', '0815.jpg', '0852.jpg', '0879.jpg', '0937.jpg', '0960.jpg'];
+
+const images = numbers.filter((item) => exceptions.indexOf(item) === -1);
+const imglen = images.length;
+
+// const newProds = [];
+// let i = 1;
+// while (newProds.length < 150) {
+//   // create array for each product
+//   // [productId, name, price, prime, imageUrl, numReviews, avgRating]
+//   newProds.push([
+//     faker.commerce.productName(),
+//     faker.commerce.price(),
+//     Math.floor(Math.random() * 2),
+//     i > imglen ? `https://d1ivqy59bo7rzu.cloudfront.net/${images[i % imglen]}` : `https://d1ivqy59bo7rzu.cloudfront.net/${images[i]}`,
+//     faker.random.number(),
+//     (Math.floor((Math.random() * 6) + 5)) / 2,
+//   ]);
+//   i += 1;
+// }
```

Comment:
you might consider doing a git checkout on the old file and breaking the new code out into its own new file. That way you can have the legacy code hanging around for reference without having huge comment blocks of legacy code.

Reply:
This is a good point. I was just trying to get to done.

Actually, this is the new file. All the legacy stuff is still in the seeder script. I'll delete the commented stuff.